src/styles/utils/opacityStyle.tsx

```diff
@@ -0,0 +1,23 @@
+import { css } from "styled-components";
+
+import { responsiveStyle } from "./responsiveStyle";
+
+export type Opacity =
```

Comment:
Lets define these in the themes folder and not here.

Reply:
I also wonder whether we want to take the same approach with opacity and go for a couple of named types. Lets ask Monika about thsi.